src/Core/Settings.cpp

```diff
@@ -5657,9 +5657,6 @@ Cluster for a shard in which current server is located
 )", BETA) \
     DECLARE(Bool, parallel_replicas_allow_in_with_subquery, true, R"(
 If true, subquery for IN will be executed on every follower replica.
-)", BETA) \
-    DECLARE(Float, parallel_replicas_single_task_marks_count_multiplier, 2, R"(
```

Comment:
Please don't remove and instead mark it as deprecated. Otherwise it will break existing queries.